### ISSUE TITLE
Modify when a user is forced to login

### ISSUE THREAD

**Issue by jgreben** [LD4P/sinopia_editor#495]
~~BLOCKED by #460~~

Currently the only requirement for authentication is the ability to upload resource templates. For the time being , make Import Resource Template button show only when logged in and remove the forced redirect to cognito login (PrivateRoute) when visiting the templates route.

In the future when people have the ability to save RDF in sinopia_server containers, that ability will also need to check for authentication status.

**Comment by jmartin-sul** [comment]
the easiest way to check for logged-in-ness is probably to wire up to redux state and look for a current session, e.g.:

https://github.com/LD4P/sinopia_editor/blob/68f4a867852397293ca9d15d4c0762c5210e16e0/src/components/App.jsx#L33
https://github.com/LD4P/sinopia_editor/blob/68f4a867852397293ca9d15d4c0762c5210e16e0/src/components/App.jsx#L55-L59

**Comment by michelleif** [comment]
@jgreben @jermnelson from user point of view, we can postpone this. we're starting with a limited set of users who will all be signing up for accounts (anyone can sign up for an account anyway!)
I think it's fine to have the entire Linked Data Editor require authentication for the time being. This will be one less thing for dev team to do. If you agree I'll move it to future.

**Comment by jermnelson** [comment]
Closing; saving RDF requires authentication in master.
